src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -0,0 +1,363 @@
+            shouldEnableMaxHeight={DeviceCapabilities.canUseTouchScreen()}
+            testID={IOURequestStepConfirmation.displayName}
+        >
+            {({safeAreaPaddingBottomStyle}) => (
+                <View style={[styles.flex1, safeAreaPaddingBottomStyle]}>
+                    <HeaderWithBackButton
+                        title={headerTitle}
+                        onBackButtonPress={navigateBack}
+                        shouldShowThreeDotsButton={requestType === CONST.IOU.REQUEST_TYPE.MANUAL}
+                        threeDotsAnchorPosition={styles.threeDotsPopoverOffsetNoCloseButton(windowWidth)}
+                        threeDotsMenuItems={[
+                            {
+                                icon: Expensicons.Receipt,
+                                text: translate('receipt.addReceipt'),
+                                onSelected: navigateToAddReceipt,
+                            },
+                        ]}
+                    />
+                    <MoneyRequestConfirmationList
+                        transaction={transaction}
+                        hasMultipleParticipants={iouType === CONST.IOU.TYPE.SPLIT}
+                        selectedParticipants={participants}
+                        iouAmount={transaction.amount}
+                        iouComment={lodashGet(transaction, 'comment.comment', '')}
+                        iouCurrencyCode={transaction.currency}
+                        iouIsBillable={transaction.billable}
+                        onToggleBillable={setBillable}
+                        iouCategory={transaction.category}
+                        iouTag={transaction.tag}
+                        onConfirm={createTransaction}
+                        onSendMoney={sendMoney}
+                        onSelectParticipant={addNewParticipant}
+                        receiptPath={receiptPath}
+                        receiptFilename={receiptFilename}
+                        iouType={iouType}
+                        reportID={reportID}
+                        isPolicyExpenseChat={isPolicyExpenseChat}
+                        // The participants can only be modified when the action is initiated from directly within a group chat and not the floating-action-button.
+                        // This is because when there is a group of people, say they are on a trip, and you have some shared expenses with some of the people,
+                        // but not all of them (maybe someone skipped out on dinner). Then it's nice to be able to select/deselect people from the group chat bill
+                        // split rather than forcing the user to create a new group, just for that expense. The reportID is empty, when the action was initiated from
+                        // the floating-action-button (since it is something that exists outside the context of a report).
+                        canModifyParticipants={!_.isEmpty(report.reportID)}
```

Comment:
The refactor has reportID for all cases so this caused #32867 which was fixed with `isFromGlobalCreate` created in #32878